lib/shift.rb

```diff
@@ -0,0 +1,57 @@
+class Shift
```

Comment:
Chose to put all information relevant to shift in one class. Encapsulated functionality.